bpython/curtsies.py

```diff
@@ -122,6 +122,9 @@ def after_suspend():
                 window.__enter__()
                 interrupting_refresh()
 
+            def get_top_usable_line():
+                return window.top_usable_row
+
```

Comment:
Thinking back the reason was probably the frustration of hopping back and forth between bpython.curtsiesfrontent.repl.py and bpython.repl.py and thinking that for poorly defined interfaces like this (dozens of methods on each, not really specced out what belonged where) that composition was easier to keep track of.